src/EnergyPlus/MixedAir.cc

```diff
@@ -4797,19 +4853,37 @@ namespace MixedAir {
 		Real64 MixEnth; // mixed air specific enthalpy [J/kg]
 		Real64 MixHumRat; // mixed air humidity ratio [kg water/kg dry air]
 		Real64 MixTemp; // mixed air temperature [C]
+		bool FirstHVACIteration;
+		int AirloopNum;
+		int CompNum;
+		static std::string CompType; //Tuned Made static
+		static std::string CompName; //Tuned Made static
+		bool Sim( true );
+		bool OAHeatCoil( false );
+		bool OACoolCoil( false );
+		bool OAHX( false );
 
 		MixNode = int( Par( 1 ) );
-		RetNode = int( Par( 2 ) );
+		RelNode = int( Par( 2 ) );
 		OANode = int( Par( 3 ) );
 		MixMassFlowRate = Par( 4 );
+
+//		FirstHVACIteration = false;
+//		if ( Par( 5 ) == 1.0 ) FirstHVACIteration = true;
+		FirstHVACIteration = ( Par( 5 ) == 1.0 );
+		AirloopNum = int( Par( 6 ) );
+
 		OAMassFlowRate = OASignal * MixMassFlowRate;
-		RecircMassFlowRate = max( MixMassFlowRate - OAMassFlowRate, 0.0 );
-		RecircEnth = Node( RetNode ).Enthalpy;
-		RecircHumRat = Node( RetNode ).HumRat;
-		MixEnth = ( RecircMassFlowRate * RecircEnth + OAMassFlowRate * Node( OANode ).Enthalpy ) / MixMassFlowRate;
-		MixHumRat = ( RecircMassFlowRate * RecircHumRat + OAMassFlowRate * Node( OANode ).HumRat ) / MixMassFlowRate;
-		MixTemp = PsyTdbFnHW( MixEnth, MixHumRat );
-		Residuum = Node( MixNode ).TempSetPoint - MixTemp;
```

Comment:
The function called using RegulaFalsi. The old method used static mixed air conditions. The new method actually simulates the OA sys components to determine the actual mixed air condition.